miniapps/gslib/findpts.cpp

```diff
@@ -169,18 +162,13 @@ int main (int argc, char *argv[])
       }
    }
 
-   Array<unsigned int> el_id_out(pts_cnt), code_out(pts_cnt),
-         task_id_out(pts_cnt);
-   Vector pos_r_out(pts_cnt * dim), dist_p_out(pts_cnt);
-
-   // Finds points stored in vxyz.
-   finder.FindPoints(vxyz, code_out, task_id_out,
-                     el_id_out, pos_r_out, dist_p_out);
-
-   // Interpolate FE function values on the found points.
+   // Find and Interpolate FE function values on the desired points.
    Vector interp_vals(pts_cnt);
-   finder.Interpolate(code_out, task_id_out, el_id_out,
-                      pos_r_out, field_vals, interp_vals);
+   // FindPoints using GSLIB and interpolate
+   FindPointsGSLIB finder;
+   finder.Interpolate(mesh, vxyz, field_vals, interp_vals);
+   Array<unsigned int> code_out = finder.GetCode();
+   Vector dist_p_out = finder.GetDist2();
 
    // Free the internal gslib data.
    finder.FreeData();
```

Comment:
Can the call of `FreeData` be moved to the destructor of `FindPointsGSLIB`? Then the data is free'd automatically when the object goes out of scope. In the parallel version, however, this still needs to be called manually before `MPI_Finalize`.

Reply:
We discussed this and had decided to leave the FreeData here for consistency between the serial and the parallel example.